nextcord/threads.py

```diff
@@ -830,3 +830,8 @@ def _from_data(self, data: ThreadMemberPayload):
     def thread(self) -> Thread:
         """:class:`Thread`: The thread this member belongs to."""
         return self.parent
+
+    @property
+    def member(self) -> Member:
```

Comment:
Shouldn't this be optional? Inline with `Guild.get_member`

Reply:
ic!